test/e2e/common/pods.go

```diff
@@ -39,13 +39,15 @@ import (
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
 var (
-	buildBackOffDuration = time.Minute
-	syncLoopFrequency    = 10 * time.Second
-	maxBackOffTolerance  = time.Duration(1.3 * float64(kubelet.MaxContainerBackOff))
+	buildBackOffDuration          = time.Minute
+	syncLoopFrequency             = 10 * time.Second
+	maxBackOffTolerance           = time.Duration(1.3 * float64(kubelet.MaxContainerBackOff))
+	maxReadyStatusUpdateTolerance = 10 * time.Second
```

Comment:
Make this 20s to lower test flake.

Reply:
I originally want to make this 5 seconds. I think kubelet's internal reconcile loop triggers every 10 seconds. I want the status update to be responsive.

Reply:
Ack. I still think this might be flaky given that the tests are run in parallel. If you want to test the latency, running a serial test would be better.

10s might be okay now, but please monitor the tests to see if it causes any flake. Actually, could you replace the `NodeConformance` tag with `[NodeFeature:]` and promote it later once it's consistently passing?